### PR TITLE
chore: Update sitemap for improving SEO

### PR DESCRIPTION
**Pull request by s-n-park**
Removed beta website from robots.txt and added additional pages to sitemap.

**Updating sitemap Process**
- Our current sitemap was not accepted to [Google Search Console](https://search.google.com/search-console/sitemaps/info-drilldown?resource_id=https:%2F%2Fnear.org%2F&sitemap=https:%2F%2Fnear.org%2Fsitemap.xml) due to unexpected format (we can just update header to <?xml ….. ?> and we can remove the closing </xml>)
- I pulled the list of top visited websites + blog pages for near.org that got >10 visitors, and manually checked ~50% to remove on-page errors or 404s. Then incorporated them into the existing sitemap generation in order to submit an updated sitemap to Google Search Console. Also included Sitemaps are only guidelines for webcrawlers and adding sites (even broken ones) cannot hurt SEO. ([source](https://developers.google.com/search/docs/crawling-indexing/sitemaps/overview?hl=en&ref_topic=4581190&visit_id=638434501191528372-1879915893&rd=1))

**Updating robots.txt**
- I removed the beta near.org website as I assumed it is no longer important for webcrawlers to list this on Google

Would love feedback on:
1. Is it okay to include many of these component pages? (ex.  near.org/devgovgigs.near/widget/...)
2. Is it okay to add in these additional pages, or would a more programmatic script make sense?
3. I'm currently unsure of the payoff for setting up a more automated process between our pages data + feeding this into the sitemap so went with a 1X adhoc update to get this started.

### CODE REVIEW

File: src/pages/sitemap.xml.tsx [hunk 1]
```diff
@@ -1,30 +1,2237 @@
...
-      </url>
-      <url>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/applications</loc>
+        <lastmod>2024-02-13</lastmod>
+    </url>
+    <url>
+        <loc>${process.env.NEXT_PUBLIC_HOSTNAME}/near/widget/ComponentDetailsPage</loc>
```
Comment (charleslavon) [inline src/pages/sitemap.xml.tsx:75]: can be removed. it has no content without proper query params

Comment (s-n-park) [reply src/pages/sitemap.xml.tsx]: done!

**Comment by s-n-park** [comment]
@charleslavon - can I get your final review here when you get a chance? 🙏

**Review by charleslavon — APPROVED** [review]
LGTM let's try it out

**Pull request merged**